src/extension.ts

```diff
@@ -81,26 +81,29 @@ export function activate(context: ExtensionContext) {
 								resolve();
 								commands.registerCommand(Commands.START_DEBUG_SESSION, async (config) => {
 									if (config.request === 'launch') {
-										if (!config.startupClass) {
-											vscode.window.showErrorMessage('Please specify startupClass in launch.json first.');
+										if (!config.mainClass) {
+											vscode.window.showErrorMessage('Please configure the main class in launch.json first.');
 											return;
-										} else if (!config.classpath) {
+										} else if (!config.classPaths || !Array.isArray(config.classPaths) || !config.classPaths.length) {
 											const params: ClasspathResolveRequestParams = {
-												startupClass: config.startupClass,
+												mainClass: config.mainClass,
 												projectName: config.projectName,
 											};
-											const classpath = await languageClient.sendRequest(ClasspathResolveRequest.type, params);
-											config.classpath = classpath;
+											config.classPaths = await languageClient.sendRequest(ClasspathResolveRequest.type, params);
+										}
+										if (!config.classPaths || !Array.isArray(config.classPaths) || !config.classPaths.length) {
+											vscode.window.showErrorMessage('Cannot resolve the class paths automatically, please configure it in launch.json first.');
+											return;
 										}
 									} else if (config.request === 'attach') {
-										if (!config.hostName || !config.port || !config.attachTimeout) {
-											vscode.window.showErrorMessage('Please specify hostName/port/attachTimeout in launch.json first.');
+										if (!config.hostName || !config.port) {
+											vscode.window.showErrorMessage('Please configure the host name and port of the attached VM in launch.json first.');
```

Comment:
configure -> specify
port of the attached VM -> the port of the remote debuggee
and remove "first"